src/features/embalm/stepContent/steps/CreateSarcohpagus.tsx

```diff
@@ -1,72 +1,88 @@
+    currentStep,
     handleCreate,
-    isUploading,
-    canCreateSarcophagus,
-    payloadTxId,
-    shardsTxId,
   } = useCreateSarcophagus();
 
-  const { approve } = useApprove();
-  const { allowance } = useAllowance();
+  const {
+    canCreateSarcophagus
+  } = useSarcophagusData();
+
+  const isCreateProcessStarted = (): boolean => {
+    return currentStep !== CreateSarcophagusStep.NOT_STARTED;
+  };
 
   return (
     <Flex
-      direction="column"
-      w="100%"
+      direction='column'
+      w='100%'
     >
       <Heading>Create Sarcophagus</Heading>
-      <Alert
-        mt={6}
-        status="info"
-      >
-        This is a temporary interface
-      </Alert>
-      <Text mt={3}>Clicking the Upload Payload button will </Text>
-      <Text ml={3}>
-        1. Encrypt the inner layer of the payload using the recipient{"'"}s public key
-      </Text>
-      <Text ml={3}>2. Encrypt the outer layer of the payload using the generated public key</Text>
-      <Text ml={3}>3. Split the outer layer private key using Shamirs Secret Sharing</Text>
-      <Text ml={3}>4. Encrypt each shard with each archaeologist{"'"}s public key</Text>
-      <Text ml={3}>5. Upload the double encrypted payload to the Arweave Bundlr</Text>
-      <Text ml={3}>6. Upload the encrypted shards to the Arweave Bundlr</Text>
-      <Text mt={3}>Steps before upload (for testing):</Text>
-      <Text ml={3}>1. Upload a payload</Text>
-      <Text ml={3}>2. Fund the Bundlr (you may need to reconnect to the Bundlr)</Text>
-      <Text ml={3}>3. Set a recipient</Text>
-      <Text ml={3}>4. Create Encryption Keypair</Text>
-      <Text mt={3}>Payload txId: {payloadTxId} </Text>
-      <Text>Shards txId: {shardsTxId} </Text>
-      <Button
-        disabled={allowance?.gt(0)}
-        onClick={() => approve()}
-      >
-        Approve Sarco Token
-      </Button>
-      <Button
-        mt={6}
-        onClick={uploadAndSetEncryptedShards}
-        isLoading={isUploading}
-        disabled={isUploading || !canCreateSarcophagus}
-      >
-        Upload Payload
-      </Button>
-      <Button
-        mt={6}
-        onClick={handleCreate}
-        isLoading={isUploading}
-        disabled={isUploading || !canCreateSarcophagus}
-      >
-        Create Sarco
-      </Button>
+      {
+        !isCreateProcessStarted() ?
+          (
+            <>
+              <SarcophagusData />
+              <Button
+                mt={6}
+                onClick={handleCreate}
+                disabled={!canCreateSarcophagus()}
+              >
+                Create Sarcophagus
+              </Button>
+            </>
+          ) :
+          (
+            <VStack
+              mt={8}
+              mb={6}
+              justify="space-between"
+              align="left"
+            >
+              <Text
+                color={currentStep === CreateSarcophagusStep.DIAL_ARCHAEOLOGISTS ? 'brand.950' : 'brand.400'}
+                align="left"
+              >
```

Comment:
This page can get a programatic refactor and move these components to a single component thats rendered via a map (or similar) once this page is updated to match the designs.

Reply:
I'm not super picky about this, but my personal preference is to not get too carried away with programatic refactors like this. If the `<Text>` element becomes too verbose, my preference would be to make a new component for it. I can explain this better in an example. 

Instead of doing this: 
```
const stageMap = [
  { stage: CreateSarcophagusStage.DIAL_ARCHAEOLOGISTS, title: "1. Dialing Archaeologists", key: "someuniquekey1" }
  { stage: CreateSarcophagusStage.UPLOAD_ENCRYPTED_SHARDS, title: "2. Uploading Encrypted Shards to Arweave", key: "someuniquekey2 }
  etc...
];

<VStack>
  {stageMap.map(stage => (
    <Text key={stage.key} prop1={} prop2={} etc>{stage.title}</Text>
  )}
</VStack>
```

I usually prefer just doing this 
```
// Make a custom component for the stage title
function StageTitle(props) {
  return <Text key={stage.key} prop1={} prop2={} etc>{props.title}</Text>
}

// Then use the component
<VStack>
  <StageTitle selected={currentStage === CreateSarcophagusStage.UPLOAD_ENCRYPTED_SHARDS}>1. Dialing Archaeologists</StageTitle>
  <StageTitle selected={currentStage === CreateSarcophagusStage.UPLOAD_ENCRYPTED_SHARDS}>2. Uploading Encrypted Shards to Arweave</StageTitle>
</VStack>
```

The reason I prefer this is because when I think to myself "Where are the stages defined?", I can easily find the components, styling, and the data associated with them in one spot without having to search for where the `stageMap` is defined. 

This is CERTAINLY open to discussion and I won't be offended if you say this is worse, this is just what I like to do because it makes a little easier for me to comprehend.

Reply:
The idea is to keep the static text in the return statement of the component where you might expect to find it, and try to put as much as we can in the `StageTitle` to keep the contents of the `VStack` down as much as possible.

Reply:
@knoll3 -- ok cool that makes sense, thanks for this context!

This component will need to be updated to match the designs, so whenever that happens these notes will come into play.